samcli/commands/build/command.py

```diff
@@ -175,6 +186,7 @@ def cli(
     config_env: str,
     hook_name: Optional[str],
     skip_prepare_infra: bool,
+    mount_with=MountMode.READ.value,
```

Comment:
Please add typing and  default value might be redundant since click sets this up automatically